### PR TITLE
Replacing Data in NLMeans Tutorial

### LINKED ISSUE #1847: Replacing Data in NLMeans Tutorial
## Description
[The tutorial of Nlmeans looks as if it is not working because of the data being used. The algorithm is not at fault, its just not suitable for that kind of data.]

### PR DESCRIPTION
**Pull request by skoudoro**
This PR fixes #1847. The tutorial of Nlmeans looks as if it is not working because of the data being used. 

This PR replaces the `sherbrooke_3shell` with `stanford_t1 hardi`

### CODE REVIEW

**Comment by arokem** [comment]
LGTM! With these changes, do the results look better than what @ShreyasFadnavis was showing in #1847?

**Comment by skoudoro** [comment]
> do the results look better than what @ShreyasFadnavis was showing in #1847?

Below the answer @arokem

# Before

Capture d’écran 2021-04-02 à 19 30 46

# After

denoised

I think the tutorial will be more explicit this way. If there is better data, I will be happy to replace it.

**Comment by ShreyasFadnavis** [comment]
Hi @skoudoro thank you for this PR 👍🏽 1 quick comment: the text in the tutorial still says `Sherbrooke Dataset` and N=4 because of a 4 Coil machine. This needs to be changed I guess. @arokem do you happen to know how many coils did the scanner for Stanford dataset have?

**Comment by arokem** [comment]
32

File: doc/examples/denoise_nlmeans.py
Comment (arokem) [inline doc/examples/denoise_nlmeans.py:31]:
```suggestion
on a 3T GE scanner with a 32 array head coil.
```

**Review by ShreyasFadnavis — APPROVED** [review]
LGTM! I can merge if anyone else doesn't have any other comments or suggestions 👍🏽

File: doc/examples/denoise_nlmeans.py
Comment (jhlegarreta) [inline doc/examples/denoise_nlmeans.py:30]: _Standford_ should be _Stanford_

**Review by jhlegarreta — COMMENTED** [review]
Noted a typo.

File: doc/examples/denoise_nlmeans.py
Comment (arokem) [reply doc/examples/denoise_nlmeans.py:30]: Good catch! 

@skoudoro : the CI failures seem unrelated to this. Do you understand what those are all about? Looks like something to do with `deform_streamlines`. Have you seen this one before?

Comment (skoudoro) [reply doc/examples/denoise_nlmeans.py]: thanks @jhlegarreta!

Yes, I have seen it before @arokem, I don't the origin yet but usually, I just restart the matrix and it passes.
For Now, I just rebase to see if the Codecov comment will appear or not.

Let's wait for the CI to finish and you can go ahead. Thanks!

Comment (ShreyasFadnavis) [reply doc/examples/denoise_nlmeans.py]: It needed 2 restarts -- everything seems green now :)

**Comment by arokem** [comment]
Let's merge it!

File: doc/examples/denoise_nlmeans.py
Comment (jhlegarreta) [reply doc/examples/denoise_nlmeans.py]: Thanks @skoudoro and all for having done this.

**Pull request merged**
